tests/test_document.py

```diff
@@ -88,6 +89,16 @@ def it_can_save_the_document_to_a_file(self, save_fixture):
         document.save(file_)
         document._part.save.assert_called_once_with(file_)
 
+    def it_provides_access_to_its_bookmarks(
+            self, body_prop_, body_, bookmarks_):
+        document = Document(None, None)
+        body_prop_.return_value = body_
+        body_.bookmarks = bookmarks_
+
+        bookmarks = document.bookmarks
+
+        assert bookmarks is bookmarks_
```

Comment:
In the past, I used to _always_ have a fixture for each test. Now I only use a test fixture when there are multiple cases (i.e. fixture needs `params=[]`). It seems like it makes it easier to have all the fixture-making right there with the test when that's possible. This doesn't apply to fixture components though (single mocks signified by a trailing underscore ('_') in their name. Those are just used directly by the test rather that a fixture.